docs/Changelog.md

```diff
@@ -10659,6 +10659,89 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrains output to boolean tensor.</dd>
 </dl>
 
+### <a name="Gather-11"></a>**Gather-11**</a>
+
+  Given `data` tensor of rank r >= 1, and `indices` tensor of rank q, gather
+  entries of the axis dimension of `data` (by default outer-most one as axis=0) indexed by `indices`, and concatenates
+  them in an output tensor of rank q + (r - 1).
```

Comment:
Could you please add some equations to explain output? For example, when `axis=0`, we have
```latex
k = indices[i_{0}, …, i_{q-1}]
output[i_{0}, …, i_{q-1}, j_{0}, …, j_{r-2}] = input[k , j_{0}, …, j_{r-2} ]
```
and
```
output[i_{0}, …, i_{q-1}, j_{0}, …, j_{r-2}] = input[j_{0}, k, j{1}, …, j_{r-2} ]
```
when `axis=1`.

Reply:
Done